cli/auth.py

```diff
@@ -69,6 +64,9 @@ def get_user_email() -> str:
     # We don't need to check verifications here,
     # because get_id_token validates the token it returns
     # with the API (this includes signature verification).
-    claims = jwt.get_unverified_claims(token)
+    try:
+        claims = jwt.get_unverified_claims(token)
+    except JWTError:
+        raise unauthenticated()
```

Comment:
In case the cached token file got corrupted, this will catch decoding errors and prompt the user to log in instead of crashing.